### DISCUSSION TITLE
How launch wezterm as external terminal from VSCode in project folder instead of home folder?

### DISCUSSION THREAD

**Discussion by ShaiAvr** [wezterm/wezterm#5975] [Q&A]
### What Operating System(s) are you running on?

macOS

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

I use Visual Studio Code on my MacBook Pro and until now I had used iTerm 2 as my terminal emulator and I had it set as the external terminal in vscode: `"terminal.external.osxExec": "iTerm.app"`. So when I press `Cmd+Shit+C` vscode launches iTerm 2 in my project folder. I recently tried out WezTerm and I liked it and decided to use it instead of iTerm 2. I changed the vscode setting to open wezterm as my external terminal: `"terminal.external.osxExec": "WezTerm.app"`. So now when I press `Cmd+Shift+C` vscode launches wezterm. However, it launches it in my home folder instead of my project folder which defeats the entire purpose of launching an external terminal from vscode. I tried searching online, but couldn't find anything useful. How can I configure vscode to use wezterm as my external terminal and launch it in the project's folder?

**Comment by BitCalSaul** [comment]
@ShaiAvr did you solve it
